File: src/Services/Device/Device.API/Model/Command.cs
using System;

namespace Device.API.Model
{
    public class Command
    {
        private String macAddress;
        private int pin;
        private bool digital;
        private float value;
    
        public string MacAddress
        {
            get => macAddress;
            set => macAddress = value;
        }

        public int Pin
        {
            get => pin;
            set => pin = value;
        }

        public bool Digital
        {
            get => digital;
            set => digital = value;
        }

        public float Value
        {
            get => value;
            set => this.value = value;
        }
    }
}

File: src/Services/Mobile.API/src/main/java/exakis/atlantis/mobileapi/dto/Command.java
package exakis.atlantis.mobileapi.dto;

public class Command {
    private String macAddress;
    private int pin;
    private boolean digital;
    private float value;

    public String getMacAddress() {
        return macAddress;
    }

    public void setMacAddress(String macAddress) {
        this.macAddress = macAddress;
    }

    public int getPin() {
        return pin;
    }

    public void setPin(int pin) {
        this.pin = pin;
    }

    public boolean isDigital() {
        return digital;
    }

    public void setDigital(boolean digital) {
        this.digital = digital;
    }

    public float getValue() {
        return value;
    }

    public void setValue(float value) {
        this.value = value;
    }
}
